### PR TITLE
Fixed high risk locations command bug and list command help message

### PR DESCRIPTION
**Pull request by hhdqirui**
1. Fixed bug: list high risk locations will change the display of person list and visit list. After fixing, the displayed person list and visit list will not change after the command is executed.
2. Fixed the list command help message.

### CODE REVIEW

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -109,7 +110,12 @@
      * Updates the filter of the filtered person list to filter by the given {@code predicate}.
      * @throws NullPointerException if {@code predicate} is null.
      */
-    void updateFilteredPersonList(Predicate<Person> predicate);
+    void updateFilteredPersonList(Predicate<? super Person> predicate);
+
+    /**
+     * Returns the current {@code Predicate} of the person filtered list
+     */
+    Optional<Predicate<? super Person>> getPersonPredicate();
```
Comment (hopinxian) [inline src/main/java/seedu/address/model/Model.java:118]: May I ask what is the significance of using <? super Person> rather than <Person> ?

Comment (hhdqirui) [reply src/main/java/seedu/address/model/Model.java:118]: The getPredicate() method of FilteredList returns Predicate<? super T> and setPredicate() method of FilteredList accept Predicate<? super T> as argument. If just keep to Predicate<Person> I need to do type casting and need to suppress the warning. So I feel it is safer to change the signature than suppressing the warning.

Comment (hopinxian) [reply src/main/java/seedu/address/model/Model.java:118]: ok got it. thank you.

**Review by hopinxian — APPROVED** [review]
LGTM

**Pull request merged**